templates/tfengine/recipes/resources.hcl

```diff
@@ -1463,6 +1463,121 @@ schema = {
         }
       }
     }
+    kubernetes_namespaces = {
```

Comment:
Can you add a note somewhere saying in order to use `kubernetes_*` resources, a `google_client_config` data resource and a `kubernetes` provider block must be specified in order for the kubernetes resource to be created under the intended cluster. See `examples/tfengine/gke_cluster.hcl` for an example?

Reply:
In the future, we could consider making a kubernetes deployment recipe which will help template some of these boilerplate.

Reply:
Done.